### ISSUE TITLE
rds file not available

### ISSUE THREAD

**Issue by Debdeb92** [chethloh/singlecelldata#1]
Hello,
I was looking at the single cell RNA-seq data ( doi: https://doi.org/10.1101/2020.07.08.192997) but the rds file is not available anymore at this link https://drive.google.com/file/d/1NiMLfB0xWj8Mibk3aP6fPxYVneuxEfYV/view?usp=sharing, is it possible to have this file?
Thanks a lot,
Best
Debora Libetti, PhD

**Comment by erzakiev** [comment]
True, it is unacceptable to store scientific datasets on google drive, one should use Zenodo for that

**Comment by erzakiev** [comment]
<img width="1018" alt="Screenshot 2023-11-21 at 16 52 23" src="https://github.com/veenstralab/chetloh_ncomm_prc2_singlecell_2021/assets/13952639/5c390efa-08a4-4de2-a1c8-7bf044bc8331">
